test/chromium/chromium.spec.ts

```diff
@@ -88,4 +91,45 @@ describe('chromium', (suite, { browserName }) => {
     // make it work with Edgium.
     expect(serverRequest.headers.intervention).toContain('feature/5718547946799104');
   });
+
+  it('should connect to an existing cdp session', async ({browserType, testWorkerIndex, browserOptions, createUserDataDir }) => {
+    const port = 9339 + testWorkerIndex;
+    const spawnedProcess = spawn(
```

Comment:
Why don't you use `chromium.launch()`? It will take care of killing the browser for you. Otherwise, you leave stray chrome when the test times out.

Reply:
Our chromium.launch specifies --remote-debugging-pipe. You can't launch headful chrome with both pipe and port. In puppeteer we had code that switched to use a websocket if port was specified, but I didn't feel like reviving it for this test.

Reply:
Passing both pipe and port works in headless. With the next roll, it should also work in headful. Let's do it.